worlds/ladx/docs/fr_Links Awakening DX.md

```diff
@@ -0,0 +1,88 @@
+
+<h3>Color Dungeon</h3>
+<p>Le Color Dungeon fait partie du mélange d'objets, et les tuniques rouges/bleues sont mélangées dans le pool d'objets. Cela signifie que la fée à la fin du Color Dungeon donne deux objets aléatoires.</p>
+<p>Pour accéder au Color Dungeon, vous avez besoin du power bracelet, et vous devez pousser les pierres tombales dans le bon ordre : "en bas, à gauche, en haut, à droite, en haut", en allant de la pierre tombale en bas à droite, à celle qui se trouve à sa gauche, au-dessus, puis à droite.</p>
+
+<h3>Bowwow</h3>
+<p>Bowwow est dans un coffre, à un endroit quelconque. Après l'avoir trouvé, il sera toujours avec vous dans le marais , mais nulle part ailleurs.</p>
+
+<h3>Éléments ajoutés</h3>
+<p>Dans votre menu de sauvegarde et de fermeture, il y a une troisième option pour retourner à votre maison. Cette option a deux utilités principales : elle accélère le jeu et empêche les softlocks (fréquents dans "l'entrance rando").</p>
+<p>Si vous avez des armes qui nécessitent des munitions (bombs, powder, arrows), un fantôme apparaîtra à l'intérieur de la Marin's house. Il vous rechargera jusqu'à 10 munitions, afin que vous n'en manquiez pas.</p>
+<p>Le flying rooster est disponible (en option) en tant qu'objet.</p>
+<p>Vous pouvez accéder à l'objet de la Bird Key cave avec le L2 Power Bracelet.</p>
+<p>La boomerang cave est maintenant un objet aléatoire par défaut (disponible après les bombs), et le boomerang est dans la randomisation d'objets.</p>
+<p>L'inventaire a été augmenté de quatre, pour tenir compte de la coexistence de plusieurs items.</p>
+
+<h3>Éléments supprimés</h3>
+<p>La mini-quête du fantôme après D4 n'apparaît jamais, sa récompense en seashell est toujours disponible.</p>
+<p>Le walrus est déplacé un peu, pour que vous puissiez accéder au désert sans prendre Marin en rendez-vous.</p>
+
+<h3>Logique</h3>
+<p>Selon vos paramètres, vous ne pouvez voler qu'après avoir trouvé l'épée, always ou never.</p>
+<p>N'oubliez pas qu'il y a deux objets dans le rafting.Vous pouvez y accéder avec le Hookshot ou les Flippers.</p>
+<p>Tuer des ennemis avec des bombs est en logique normal. Vous pouvez passer en logique casual si vous ne le souhaitez pas.</p>
+<p>Le D7 perturbe certaines personnes, mais en descendant dans les fosses du 2ème étage, vous pouvez accéder à la quasi-totalité de ce donjon, même sans plume ni bracelet de puissance.</p>
+
+<h3>Technique</h3>
+<p>Le toadstool et la powder magic étaient auparavant le même type d'objet. LADXR les transforme en deux objets que vous pouvez avoir en même temps. 4 emplacements d'objets supplémentaires dans votre inventaire ont été ajoutés pour prendre en charge cet objet supplémentaire et avoir la possibilité de posséder le boomerang.</p>
+<p>Le glitch selon lequel la slime key est en fait une 6ème golden leaf est corrigé, et les feuilles d'or peuvent être collectées à côté de la slime key.</p>
```

Comment:
```suggestion
<p>Le **glitch** selon lequel la **slime key** est en fait une 6ème **golden leaf** est corrigé, et les feuilles d'or peuvent être collectées à côté de la **slime key**.</p>
```

Reply:
same thing here

Reply:
[le glitch je suis ](https://www.larousse.fr/dictionnaires/anglais-francais/glitch/583673)
le mot glitch existe en français donc pas de gras
item info voir plus haut^^

Reply:
Okay thank you. I’m not an expert.